src/ctia/bulk/core.clj

```diff
@@ -157,6 +157,28 @@
     [:ctia :store :bulk-refresh]
     "false"))
 
+(s/defn create-asset-ref-entities
+  "Creates AssetMapping and AssetProperties entities, rectifying transient IDs in the process"
+  [bulk temp-ids login refresh
+   services :- APIHandlerServices]
+  (let [bulk' (select-keys bulk [:asset_mappings :asset_properties])
+        ;; replace :asset_ref fields with non-transient IDs
+        ents (zipmap
+              (keys bulk')
+              (->> bulk'
+                   vals
+                   (map
+                    (partial
+                     map #(assoc % :asset_ref (get temp-ids (:asset_ref %)))))))]
```

Comment:
perhaps we should add some check here in case the entities was not created for any reason.

Reply:
Absolutely. Will be addressed within the scope of https://github.com/threatgrid/iroh/issues/4917

Reply:
> long-id ==> keep it (we do not check that the corresponding asset exists, it's the responsability of the client)

Can you elaborate on that? So you think we'd have situations where AssetMappings/AssetProperties would be created ahead-of-time (before Assets)? And then the client would have to create Assets later?
If we allow that to happen, wouldn't that increase entropy and allow inconsistent states? I guess my assumptions in threatgrid/iroh#4917 are slightly not correct?

Reply:
No I mean that asset-mappings and asset-properties could be pushed later.
Like I mentioned:
> For instance a device while be assigned new ips, and we while have new sightings observed and thus mapped to it.

So in this example here could be the life cycle:
- at t1, A bundle is pushed with an `asset` to represent a machine and an `asset-mapping` to map it to an ip.
- at t2 the machine ip changes. 
- at t3  a bundle is pushed with an `asset-mapping` to map it to an ip.

It could also happen to any other mapped observable and any other mapped properties.